Add reference checks to the hiring process (#4205)
FlowFuse/website: src/handbook/peopleops/hiring/reference-check.md

---
navTitle: Reference Check
---

Reference checks are a much needed final stage in the hiring process, conducted
after the interview stages but before extending an offer. These conversations
provide valuable insights into a candidate's past performance, work style, and
potential fit with FlowFuse.

## When to Conduct Reference Checks

Reference checks should be conducted after all interviews are complete and the
hiring team has identified a preferred candidate. We request 3 professional

> references from the candidate, preferably including:

Reviewer: ```suggestion
Reference checks should be conducted after all interviews are complete and the hiring team has identified a preferred candidate. We typically request three professional references from the candidate, preferably including:
```

- A former direct manager
- A peer or colleague
- A direct report (if applicable for the role)

FlowFuse will conduct 1 or 2 reference check calls, typically prioritizing

> conversations with former managers who can speak to the candidate's performance.

Reviewer: ```suggestion
We request references to ensure a range of perspectives. In cases where providing three references is not feasible (for example, candidates who are currently employed or early in their career), the hiring team may proceed with fewer references and treat this as contextual information rather than a blocker.
```

## Structure of a Reference Check Call

> The reference check is typically a 20-30 minute conversation conducted over the phone or video call. The structure should be:

Reviewer: ```suggestion
The reference check is typically a 20–30 minute conversation conducted over the phone or video call. Live conversations are preferred over async forms to allow for nuance, follow-up questions, and contextual understanding. The flow should be:
```

1. Introduce yourself and FlowFuse, and explain the purpose of the call.
2. Confirm the reference's relationship to the candidate and time worked together.
3. Ask the [reference check questions](#reference-check-questions).
4. Thank the reference for their time and insights.

## Reference Check Questions

> During the reference check, aim to understand:

Reviewer: ```suggestion
These questions are intended to surface working style, strengths, and growth areas to support onboarding and early management. When speaking with peers or colleagues, ask for concrete examples and specific situations rather than general impressions.

During the reference check, aim to understand:
```

1. Their working relationship; In what capacity did they collaborate? How long did you work together, and in what time period?
1. What's their one trait you remember really well?
1. How does the candidate prefer to work?
   - Does this person work best in fully autonomous settings, or as part of a team?
1. How would you place this person compared to everyone you've worked with? (top 0.1%, 1%, 10%, ...)
1. What should the candidate's new manager / team know to help them be successful in their next role?

> 1. Who else should the company speak with (if you're looking to run backdoor references).

Reviewer: ```suggestion
1. Is there anyone else who might be helpful to speak with, if the hiring team chooses to seek additional context?
```

> 1. Who else should the company speak with (if you're looking to run backdoor references).

Reviewer: ```suggestion
1. Can you describe how you worked with [Candidate Name] and how long you worked together?
1. What is one trait or quality that stands out most to you?
1. How does [Candidate Name] typically prefer to work day to day?
   - For example, do they tend to do best in highly autonomous environments, or in more collaborative team settings?
1. Compared to others you’ve worked with in similar roles, how does [Candidate Name] stand out?
1. What would you want [Candidate Name]’s new manager or team to know to help them be successful?
1. (Optional) Is there anyone else you think might be helpful for us to speak with if we want additional context?
```
